services/eth_gateway_mock.go

```diff
@@ -0,0 +1,23 @@
+package services
+
+import (
+	"github.com/oysterprotocol/brokernode/models"
+)
+
+var EthMock Eth
+
+func init() {
+	SetUpMock()
+}
+
+func SetUpMock() {
+
+	EthMock = Eth{
+		ClaimPRLs: func([]models.CompletedUpload) error {
+			return nil
+		},
+		SendGas: func([]models.CompletedUpload) error {
+			return nil
+		},
```

Comment:
Add generic mocks for other methods here, once we know for sure the params and return values.